<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productadder</class>
 <widget class="QDialog" name="productadder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color: rgb(34, 40, 49);
	
}



QDialog
{
	background-color: rgb(238, 238, 238);
}



QLabel
{
	background-color:rgb(238, 238, 238);
	color: rgb(34, 40, 49);
	font: 11pt &quot;DejaVu Sans Mono&quot;;
}

QLineEdit
{

	color:rgb(238, 238, 238);
	background-color: rgb(34, 40, 49);
	border-radius: 9px;
	font: 9pt &quot;DejaVu Sans Mono&quot;;

}


QPushButton
{
	border-radius: 12px;
	background-color: rgb(255, 211, 105);
	color: rgb(34, 40, 49);
	font: 700 11pt &quot;DejaVu Sans Mono&quot;;
	
	
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>0</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 18pt &quot;Ubuntu&quot;;
background-color: rgb(34, 40, 49);
color:rgb(238, 238, 238);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Products Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addProduct">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Stock</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Unit Price</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearScreen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorDisplay">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="stockIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="unitIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="addedSucessfully">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>unitIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>stockIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>codeIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>nameIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>errorDisplay</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textEdited(QString)</signal>
   <receiver>errorDisplay</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorDisplay</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>addedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>addedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>659</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>addedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>652</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
